src/SurfaceForceAndMomentAlgorithm.C

```diff
@@ -54,7 +55,8 @@ SurfaceForceAndMomentAlgorithm::SurfaceForceAndMomentAlgorithm(
   const std::string &outputFileName,
   const int &frequency,
   const std::vector<double > &parameters,
-  const bool &useShifted)
+  const bool &useShifted,
+  const WallBoundaryConditionData &wallBCData)
```

Comment:
I feel that this capability already exists in SurfaceForceAndMomentWallFunctionAlgorithm. This class shouldn't be modified.

Reply:
Could you explain the difference in the methods used to calculate uTau, tauWall (e.g. lambda vs. no lambda), and yplus in SurfaceForceAndMoment vs. SurfaceForceAndMomentWallFunction?